dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -19,6 +21,10 @@ export const DynamicFast = ({ trails }: Props) => {
 			<UL direction="row" padBottom={true}>
 				<LI padSides={true} percentage="75%">
 					<Card
+						frontPalette={
+							containerPalette &&
+							decideFrontPalette(containerPalette)
+						}
```

Comment:
Worth extracting as a `const` at the top of this component?

Reply:
I've refactored away from this pattern now in favour of passing the type down